lxd/daemon.go

```diff
@@ -765,15 +765,15 @@ func (d *Daemon) Init() error {
 		return err
 	}
 
-	// 
+	//
```

Comment:
So hmm, it's nice to remove the trailing white spaces, but shouldn't the empty comment lines also be removed? :)
There are a whole bunch more occurrences of this in this commit, can we just remove all those leading empty comment lines that were introduced with this branch?